app/components/onboarding-message/template.hbs

```diff
@@ -0,0 +1,27 @@
+{{#liquid-if showOnboardingMessage}}
+<div class="onboarding-message-container">
+  <a href="#" class="onboarding-close" {{action 'hideOnboardMessage'}}>{{svg-jar "close"}} </a>
+  <div class="text-and-note">
+    <div class="music-note-container">
+      {{svg-jar "noun-music" class="music-note" }}
+    </div>
+    <div class="onboarding-message-text-container">
+      <div class="first-half">
+        <h3 class="onboarding-title" data-test-element="title"> You're invited... </h3>
```

Comment:
as a general cleanliness note, you can remove the spaces around `You're invited...` and in other places (such as line 3 above, before the `<\a>`